client/src/pages/DetailsPage/item-details.module.scss

```
@@ -2,26 +2,179 @@
 
 
 .itemDetails {
-    .testi1 {
-        background-color: blue;
+    width: 90vw;
+    margin: 0 auto;
+    text-align: center;
 
-        image {
-            width: 80vw;
-            max-width: 960px;
+    .upperPart {
+        display: flex;
+        margin-top: 4%;
+
+        .pageTitle {
+            font-size: $fz-small-5;
+            color: $fc-5;
+        }
+
+        img {
+            width: 10%;
+
+        }
+    }
+
+    .imageSection {
+        margin-top: 4%;
+
+        img {
+            width: 90%;
+            margin-bottom: 3%;
+            border-radius: 10px;
+        }
+
+        .quantityIndicator {
+            font-size: $fz-small-2;
+            background-color: $bc-1;
+            width: 10%;
+            padding: 2% 7%;
+            margin: 0 auto 3% auto;
+            border-radius: 7px;
+            color: $fc-2;
+        }
+
+        .priceIndicator {
+            color: $fc-5;
+            margin-bottom: 1.5%;
         }
 
+        .nameIndicator {
+            font-size: $fz-small-4;
+            color: $fc-5;
+            margin-bottom: 1.5%;
+        }
     }
 
-    .quantityIndicator {
-        background-color: green;
-        width: 20vw;
-        height: 10vw;
+    .ratingSection {
+        display: flex;
+        justify-content: center;
+        margin-bottom: 5%;
+
+        img {
+            width: $fz-small-4;
+            margin-right: 1.4%;
+        }
+
+        .ratingIndicator {
+            font-size: $fz-small-4;
+            color: $fc-4;
+            margin-right: 1%;
+        }
+
+        .reviewIndicator {
+            font-size: $fz-small-4;
+            color: $fc-4;
+        }
     }
 
-    .quantitySelection {
-        font-family: $ff-roboto;
-        font-size: 5rem;
-        background-color: green;
+    hr {
+        color: #c0c0c0;
     }
 
+    .quantitySection {
+        margin-top: 4%;
+
+        .quantityTitle {
+            font-size: $fz-small-4;
+            color: $fc-4;
+            margin-bottom: 3%;
+        }
+
+        .selectQuantity {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            margin-bottom: 6%;
+
+            .minusQuantity,
+            .plusQuantity {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                font-size: $fz-large-1;
+                padding: 5.4%;
+                width: 2rem;
+                height: 2rem;
+                border-radius: 50%;
```

Das ist soweit erstmal okay. Wenn du hier aber einen button hast der bisschen länger ist wird der Oval gestreckt. Das kannst du unterbinden indem du einfach einen sehr hohen Wert für Border-Radius nimmst.
```suggestion
                border-radius: 999vw;
```

---

done! danke für den tipp!